packages/navigation-menu/examples/src/navigation-menu/pages/demo/demo.page.html

```diff
@@ -0,0 +1,112 @@
+<h2 class="h3 u-margin-bottom">Navigation Menu</h2>
```

Comment:
the demo pages appear with extra scrollbars for an unclear reason (observed issue in brave and chromium)
![scrollbars](https://user-images.githubusercontent.com/2686747/72718772-dc836600-3b76-11ea-8e8e-9d71681f503c.png)